challenges/objects-arrays.js

```diff
@@ -54,12 +88,26 @@ The resulting contact information should have a space between the first name and
 Name email@example.com
 
 Log the result of your new array. */
+
 const contactInfo = [];
+
+for (let i = 0; i <= index ; i++) {
+
+  let nameEmail = `${graduates[i].first_name}${'s'} email is: ${graduates[i].email}`;
```

Comment:
Follow the instruction of how to display the information...

The resulting contact information should have a space between the first name and the email information like this: 
``` 

Name email@example.com

```